### PR TITLE
feat: LIVE-6280 allow device pairing with open app on LLM (new selector)

### PR DESCRIPTION
**Pull request by juan-cortes**
### 📝 Description

The new device selector (the old one also fails) fails to complete a pairing flow when there's an open application on the device. The reason for this is the use of the `getVersion` apdu instead of the `getAppAndVersion` apdu since one is available only on the dashboard and the other is available virtually everywhere.

### ❓ Context

- **Impacted projects**: `ledger-live-common` 
- **Linked resource(s)**: `https://ledgerhq.atlassian.net/browse/LIVE-6280` 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo
No demo

### 🚀 Expectations to reach
- With the new device selector feature flag enabled
- Make sure your device is not paired with your phone, delete if from Live if necessary
- Enter BTC/ETH or some other application on your nano
- Attempt to pair the device on LLM

It is expected that the pairing will succeed (if you agree to the code of course). Entering the manager afterwards should automatically quit the app and access the screen.  Doing the same without the feature flag enabled should fail since the logic is relying on `getVersion` which fails if not on the dashboard.

### CODE REVIEW

File: libs/ledger-live-common/src/ble/hooks/useBleDevicePairing.ts [hunk 1]
```diff
@@ -31,10 +30,10 @@ export const useBleDevicePairing = ({
   const [pairingError, setPairingError] = useState<PairingError>(null);
 
   useEffect(() => {
-    const requestObservable = withDevice(deviceId)(t => from(getVersion(t))).pipe(first());
+    const requestObservable = withDevice(deviceId)(t => from(getAppAndVersion(t))).pipe(first());
```
Comment (alexandremgo) [inline libs/ledger-live-common/src/ble/hooks/useBleDevicePairing.ts:33]: It's a great idea ! Though i think you need to update `libs/ledger-live-common/src/ble/hooks/useBleDevicePairing.test.ts` so it mocks `getAppAndVersion` now (and not `getVersion`) ;)

Comment (juan-cortes) [reply libs/ledger-live-common/src/ble/hooks/useBleDevicePairing.ts:33]: Good catch, pushed.

**Review by alexandremgo — APPROVED** [review]
Great fix, thx ! 🐬

**Comment by juan-cortes** [comment]
Not doable for now until we figure out something involving FW
https://ledger.slack.com/archives/C02KVKX8GCF/p1687504327915769

**Pull request closed without merging**